card/mod/05_email/set/all/notify.rb

```diff
@@ -68,7 +68,7 @@ def silent_change?
 end
 
 def notable_change?
-  !silent_change? && !supercard && current_act &&
+  !silent_change? && !supercard && current_act && current_act.card_id == id &&
```

Comment:
If I understand this, you're saying that only changes to the current act card are notable.  I don't think that's what is intended at all.